mobile/flutter_app/lib/widgets/dashboard/dashboardTop.dart

```diff
@@ -1,65 +1,75 @@
+      return MediaQuery.of(context).size.width * percent;
     }
 
-		return Container(
-			color: Color(0xff4B89AC),
-			child: Stack(
-				alignment: Alignment.topCenter,
-				children: [
-					Container(
-            height: _viewWidth(0.40),
-            margin: const EdgeInsets.all( 0),
-            padding: const EdgeInsets.all(0),
-            child: Image.asset("assets/images/corner_circles.png",
-              width: double.infinity,
-              alignment: Alignment.topLeft,
-           )
-				 	),
-					Container(
-						width: 100,
-						height: 100,
-						margin: const EdgeInsets.only(top: 100),
-						decoration: BoxDecoration(
-							shape: BoxShape.circle,
-							image: DecorationImage(
-								image: NetworkImage("https://www.pixsy.com/wp-content/uploads/2021/04/ben-sweet-2LowviVHZ-E-unsplash-1.jpeg"),
-								fit: BoxFit.fitHeight
-							),
-						),
-					),
-					Padding(
-						padding: const EdgeInsets.only(top: 210),
-						child: Text(
-							'Welcome, Alan Pisano',
-							style: const TextStyle(
-								color: Color(0xffE4FCF9),
-								fontFamily: 'Roboto',
-								fontSize: 18,
-								fontWeight: FontWeight.w500,
-								decoration: TextDecoration.none
-							),
-						),
-					)
-				],
-			),
-			height: 250
-		);
-	}
+    return Container(
+        color: Color(0xff4B89AC),
+        child: Stack(
+          alignment: Alignment.topCenter,
+          children: [
+            Container(
+                height: _viewWidth(0.40),
+                margin: const EdgeInsets.all(0),
+                padding: const EdgeInsets.all(0),
+                child: Image.asset(
+                  "assets/images/corner_circles.png",
+                  width: double.infinity,
+                  alignment: Alignment.topLeft,
+                )),
+            Container(
+              width: 100,
+              height: 100,
+              margin: const EdgeInsets.only(top: 100),
+              decoration: BoxDecoration(
+                shape: BoxShape.circle,
+                image: DecorationImage(
+                    image: NetworkImage(
+                        "https://www.pixsy.com/wp-content/uploads/2021/04/ben-sweet-2LowviVHZ-E-unsplash-1.jpeg"),
+                    fit: BoxFit.fitHeight),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.only(top: 210),
+              child: GestureDetector(
+                onTap: () {
+                  signOutHandler(context);
+                },
+                child: Text(
+                  'Welcome, Alan Pisano',
+                  style: const TextStyle(
+                      color: Color(0xffE4FCF9),
+                      fontFamily: 'Roboto',
+                      fontSize: 18,
+                      fontWeight: FontWeight.w500,
+                      decoration: TextDecoration.none),
+                ),
+              ),
+            ),
+          ],
+        ),
+        height: 250);
+  }
```

Comment:
Could you summarize what changes you made here please?

Reply:
I think it's just my code formatter formatting the spaces... i didn't change anything i believe 😅

Reply:
Oh, added the gesture signout on the welcome text...

Reply:
Could you add a comment to take out the gesturehandler once we add in a logout button?
// TODO: Take out GestureHandler after logout button is added in

Reply:
Done